content/en/examples/controllers/frontend.yaml

```diff
@@ -18,4 +18,4 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
+        image: us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v3
```

Comment:
```suggestion
        image: us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5
```

Reply:
Only one tag v5 exists and tested v3 will also not work as not available in repository.